cdk/stack/ynr.py

```diff
@@ -262,6 +263,26 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         Tags.of(web_service).add("role", "web")
         Tags.of(worker_service).add("role", "worker")
 
+        s3_resources = ["arn:aws:s3:::ynr-*"]
+        if self.dc_environment == "production":
+            s3_resources = [
+                "arn:aws:s3:::ynr-*",
+                "arn:aws:s3:::public-sopns",
+                "arn:aws:s3:::public-sopns/*",
+                "arn:aws:s3:::static-candidates.democracyclub.org.uk",
+                "arn:aws:s3:::static-candidates.democracyclub.org.uk/*",
+            ]
```

Comment:
Refs https://app.asana.com/1/1204880536137786/project/1204880927741389/task/1211210831470667?focus=true
If we want to allow cross-account access by policy, we also have to allow the outgoing requests here.
I've checked this by deploying this policy on dev and making sure I can list bucket contents on those two buckets from a container